### PR TITLE
Minor refinements for share menu

### PR DESCRIPTION
**Pull request by kandre**
Issue #2424

I made some minor changes in the share menu according the usability session.

@davidoesch @p1d1d1 could you please complete traductions (re3) in your respective language ?

### CODE REVIEW

File: src/components/share/partials/share.html [hunk 1]
```diff
@@ -51,9 +49,6 @@
     <span ng-hide="showMore" translate>share_more</span>
   </div>
   <div class="ga-share-embed" ng-show="showMore">
-    <label>
-      <span translate>embed_map</span>:
```
Comment (oterral) [inline src/components/share/partials/share.html]: Remove embed_map from translations files  if it's not use elsewhere

File: src/locales/rm.json [hunk 2]
```diff
@@ -248,12 +248,12 @@
 	"share_file_disclaimer": "Ihre Zeichnung wird  automatisch für ein Jahr gespeichert. Durch  die Nutzung  dieses Dienstes stimmen Sie den  <a href='http://www.geo.admin.ch/internet/geoportal/de/home/geoadmin/contact.html#drawing ' target='_blank'>Nutzungsbedingungen</a> zu.",
 	"share_file_link_descr_admin": "",
 	"share_file_link_title_admin": "Sie können Ihre Zeichnung später über den folgenden Link editieren: ",
-	"share_less": "Pli ...",
+	"share_less": "Code pour intégrer la carte :",
```
Comment (oterral) [inline src/locales/rm.json]: Nicht sehr Deutsch

Comment (oterral) [inline src/locales/rm.json]: Sorry it's rumantsch:)

File: src/components/share/partials/share.html (hunk 1 above)
Comment (kandre) [inline src/components/share/partials/share.html]: It is used in the modal viewer

**Comment by davidoesch** [comment]
Instead of more options analog to youtube
FR "Intégrer"
IT "Codice da incorporare"
EN "Embed"
DE "Einbetten"

**Comment by kandre** [comment]
Thanks a lot !

**Comment by cedricmoullet** [comment]
@kandre is it ready ?

**Comment by kandre** [comment]
@davidoesch in last meetin we talk about changing share_disclaimer
from
'Vous pouvez intégrer la carte dans votre site web ou votre blog'
to
'Ce code vous permet d’intégrer la carte dans votre site web ou votre blog'

Could please do this last translation in our re3 doc ?

**Comment by kandre** [comment]
@davidoesch about your message of writing just 'embed' as youtube, lets try like that and see with uservalue meeting

**Comment by cedricmoullet** [comment]
jenkins test

**Comment by kandre** [comment]
I made the translation by myself. Ready for review/merge

**Comment by davidoesch** [comment]
Use the translation table and the text above.... This will work. We try with less text as possible

**Comment by kandre** [comment]
yes but we discuss to change a bit the disclaimer also

**Comment by davidoesch** [comment]
Nope you don't change the disclaimer. This one had been approved by steering committee and legal....

**Comment by kandre** [comment]
ok I see, I put it back to the original state

**Pull request merged**